week-3/schedule.md

```diff
@@ -0,0 +1,55 @@
+# Testing Week Schedule
+
+## Day 1
+
+| Time    | Activity                               | Learning outcomes |
+| ------- | -------------------------------------- | ----------------- |
+| 10:00   | Intro presentation                     |                   |
+| 10:30   | [Build a testing library](testing-lib) | Testing           |
+| 11:00   | Unit testing workshop                  | Testing           |
+| _13:00_ | _Lunch_                                |                   |
+| 14:00   | Testing an actual thing workshop (tbc) | Testing           |
+| 16:00   | Tech for Better                        |                   |
+
+[testing-lib]: https://github.com/oliverjam/learn-testing/
+
+## Day 2
+
+| Time    | Activity                | Learning outcomes                      |
+| ------- | ----------------------- | -------------------------------------- |
+| 10:00   | Colour palette workshop | CSS variables, colour palette          |
+| 11:00   | TDD array methods       | Test-Driven-Development, array methods |
+| _13:00_ | _Lunch_                 |                                        |
+| 14:00   | Project intro           |                                        |
+| 14:15   | Research                |                                        |
+| 16:30   | Research presentations  |                                        |
+
+## Day 3
+
+| Time    | Activity            | Learning outcomes           |
+| ------- | ------------------- | --------------------------- |
+| 10:00   | DOM Manipulation MC | add/modify/remove DOM nodes |
```

Comment:
I think it's a good idea for the Wednesday morning Morning Challenge to be relevant to the project they're about to do. i.e. here they're about to build a to-do list with lots of DOM manipulation, so lets get good practices in their head right before they start.